src/selectors/selectedFamily.js

```diff
@@ -1,12 +1,33 @@
 import { createSelector, createSelectorCreator, defaultMemoize } from 'reselect';
 import * as _ from 'lodash';
 
-const getSelectedFamily = (state) => {
+// The naming structure here needs to be cleaned up; 
```

Comment:
Just in terms of the selectors or the whole redux store naming conventions?

Reply:
The selector names; the one for the tree structure especially, since it's not just a tips collection.